src/hotspot/share/interpreter/bytecodeTracer.cpp

```diff
@@ -529,7 +524,7 @@ void BytecodePrinter::print_attributes(int bci, outputStream* st) {
             ClassPrinter::has_mode(_flags, ClassPrinter::PRINT_METHOD_HANDLE)) {
           assert(is_linked(), "invokehandle is only in rewritten methods");
           assert(cpcache_index >= 0, "must be");
-          print_cpcache_entry(cpcache_index, st);
+          print_field_or_method(cp_index, st);
```

Comment:
I don't understand this code very well but it looks like this change means `print_field_or_method` gets called twice when we have an `invokehandle` bytecode, passing `cp_index` both times. Is that intended?

Reply:
This comment is still left to address, Matias.